### PR TITLE
New cmd keywords for related telemetry and remove legacy snmp

### LINKED ISSUE #403: Associated Telemetry Points with Commands
Add command keywords to reference telemetry points.
This will allow us to show associated telemetry in CmdSender.

### PR DESCRIPTION
**Pull request by ryanmelt**
closes #403

### CODE REVIEW

File: openc3/lib/openc3/packets/packet_config.rb [hunk 1]
```diff
@@ -214,7 +214,8 @@ def process_file(filename, process_target_name)
               'PARAMETER', 'ID_ITEM', 'ID_PARAMETER', 'ARRAY_ITEM', 'ARRAY_PARAMETER', 'APPEND_ITEM',\
               'APPEND_PARAMETER', 'APPEND_ID_ITEM', 'APPEND_ID_PARAMETER', 'APPEND_ARRAY_ITEM',\
               'APPEND_ARRAY_PARAMETER', 'ALLOW_SHORT', 'HAZARDOUS', 'PROCESSOR', 'META',\
-              'DISABLE_MESSAGES', 'HIDDEN', 'DISABLED', 'ACCESSOR', 'TEMPLATE', 'TEMPLATE_FILE'
+              'DISABLE_MESSAGES', 'HIDDEN', 'DISABLED', 'ACCESSOR', 'TEMPLATE', 'TEMPLATE_FILE',\
+              'RESPONSE', 'ERROR_RESPONSE', 'SCREEN', 'RELATED_ITEM'
```
Comment (jmthomas) [inline openc3/lib/openc3/packets/packet_config.rb:218]: You need to document all these keywords!

File: openc3/lib/openc3/packets/packet_config.rb [hunk 2]
```diff
@@ -460,7 +461,30 @@ def process_current_packet(parser, keyword, params)
...
+
+      when 'RESPONSE'
+        usage = "#{keyword} <Target Name> <Packet Name>"
+        parser.verify_num_parameters(2, 2, usage)
+        @current_packet.response = [params[0].upcase, params[1].upcase]
+
+      when 'ERROR_RESPONSE'
+        usage = "#{keyword} <Target Name> <Packet Name>"
+        parser.verify_num_parameters(2, 2, usage)
+        @current_packet.error_response = [params[0].upcase, params[1].upcase]
```
Comment (jmthomas) [inline openc3/lib/openc3/packets/packet_config.rb:473]: How will RESPONSE and ERROR_RESPONSE be used?

**Review by jmthomas — CHANGES_REQUESTED** [review]

File: openc3-cosmos-init/plugins/packages/openc3-cosmos-demo/targets/INST/cmd_tlm/inst_cmds.txt [hunk 3]
```diff
@@ -9,19 +9,24 @@ COMMAND <%= target_name %> COLLECT BIG_ENDIAN "Starts a collect on the <%= targe
...
   <%= render "_ccsds_cmd.txt", locals: {id: 3} %>
+  RELATED_ITEM <%= target_name %> HEALTH_STATUS COLLECTS
 
 COMMAND <%= target_name %> SETPARAMS BIG_ENDIAN "Sets numbered parameters"
   <%= render "_ccsds_cmd.txt", locals: {id: 4} %>
   # ERB syntax:
   <% (1..5).each do |i| %>
     APPEND_PARAMETER VALUE<%= i %> 16 UINT 0 5 1 "Value <%= i %> setting"
+    RELATED_ITEM <%= target_name %> PARAMS VALUE<%= i %>
   <% end %>
```
[the remark this answers is not in the document]
Comment (ryanmelt) [reply openc3-cosmos-init/plugins/packages/openc3-cosmos-demo/targets/INST/cmd_tlm/inst_cmds.txt:30]: I'll add both.  SCREEN wins if present.

File: openc3/lib/openc3/packets/packet_config.rb (hunk 1 above)
Comment (ryanmelt) [reply openc3/lib/openc3/packets/packet_config.rb:218]: Yep.

File: openc3/lib/openc3/packets/packet_config.rb (hunk 2 above)
Comment (ryanmelt) [reply openc3/lib/openc3/packets/packet_config.rb:473]: This will be the standard way to build command/response patterns.  I'm using it with SNMP

**Review by jmthomas — APPROVED** [review]

**Pull request merged**